docs/sdk/web-backend.mdx

```diff
@@ -0,0 +1,290 @@
+---
+title: "@web3auth/node-sdk"
+displayed_sidebar: sdk
+description: "@web3auth/node-sdk package | Documentation - Web3Auth"
+---
+
+import ChainConfig from "@site/src/common/sdk/web/_chain-config.mdx";
+import Web3AuthOptions from "@site/src/common/sdk/web/_web3authcore-options.mdx";
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+---
+
+## [`@web3auth/node-sdk`](https://npmjs.com/package/@web3auth/node-sdk)
+
+Web3Auth's `@web3auth/node-sdk` backend SDK is a simple and easy to use SDK to be used in your Node.js backend to get the same experience of Web3Auth
+frontend SDKs. One thing to note here is that, with this you get semi-custodiality.
```

Comment:
explain why this is semi custodial, maybe an architecture diagram will be a good addition. you can copy the one key flow diagram to edit the package name here. People need to know properly how this works.